packages/lib/src/vis/hooks.ts

```diff
@@ -104,3 +104,36 @@ export function useCustomColors(
 
   return [colors, refCallback];
 }
+
+export function useRaycasterThreshold(
+  lineOrPoints: 'Line' | 'Points',
+  desiredThreshold: number
+) {
+  const raycaster = useThree((state) => state.raycaster);
+  const zoomVector = useThree((state) => state.camera.scale);
+
+  useEffect(() => {
+    const params = raycaster.params[lineOrPoints];
+    const oldThreshold = params ? params.threshold : 1;
+
+    if (params) {
+      params.threshold = desiredThreshold;
+    }
```

Comment:
Annoyingly, 
```
if(raycaster.params[lineOrPoints]) {
  raycaster.params[lineOrPoints].threshold
}
```
raises a TS error as TS cannot infer that `raycaster.params[lineOrPoints]` is defined in the `if` branch so I have to resort to an intermediate variable